code/modules/projectiles/updated_projectiles/guns/shotguns.dm

```diff
@@ -92,6 +92,11 @@ can cause issues with ammo types getting mixed up during the burst.
 	new_handful.generate_handful(selection, "12g", 5, 1, /obj/item/weapon/gun/shotgun)
 	return new_handful
 
+/obj/item/weapon/gun/shotgun/pump/bolt/retrieve_shell(selection)
+	var/obj/item/ammo_magazine/handful/new_handful = new /obj/item/ammo_magazine/handful()
+	new_handful.generate_handful(selection, "7.62x54mmR", 5, 1, /obj/item/weapon/gun/shotgun)
```

Comment:
if you're doing this you may as well setup a subtype of handful that has the correct names/iconstate calibre etc.